docker-compose.yml

```diff
@@ -118,6 +118,6 @@ services:
       COURT_DATA_ADAPTOR_API_SECRET: test-app-secret
       CASEWORKER_PASSWORD: caseworker-password
       MANAGER_PASSWORD: manager-password
-      SHOW_APPEALS: 'true'
```

Comment:
Could you double-check that this is not used in other part of the UI?

Reply:
Yup, we removed this in favour of the per-user appeals feature flags.